app/assets/stylesheets/components/internal_link.scss

```diff
@@ -10,7 +10,7 @@
       content: "\f0c1";
       font-family: "Font Awesome 5 Free";
       font-size: .75em;
-      left: -1.2em;
+      left: -1.6em;
```

Comment:
Is this from a previous PR?

Reply:
yeah, i dont know why its tagged along!! do i have to delete this from my 'forked/local' branch?! i was quite surprised to see this here as well!! please advise, thanks :)

Reply:
Did you branch off the `fixed-curriculum-header-statements-on-hover-effect-visual` branch instead of main by any chance?

Reply:
probably so!! again, 'lesson is learned', from now on 'new branch should come from main - double check!!' so do i delete this existing branch start a fresh, along with a new PR?! @KevinMulhern